app/post.go

```diff
@@ -1030,10 +1029,5 @@ func (a *App) ImageProxyRemover() (f func(string) string) {
 }
 
 func (a *App) MaxPostSize() int {
-	result := <-a.Srv.Store.Post().GetMaxPostSize()
-	if result.Err != nil {
-		mlog.Error(fmt.Sprint(result.Err))
-		return model.POST_MESSAGE_MAX_RUNES_V1
```

Comment:
Previously, the code checks for error, if there is an error, its logs and returns `model.POST_MESSAGE_MAX_RUNES_V1`. But the GetMaxPostSize() never produces an error in result, So `result.Err != nil` is unreachable.  

In new case, we can check if output is 0(assuming 0 is incorrect output) then return `model.POST_MESSAGE_MAX_RUNES_V1` and can probably update the test suite to fix the build.

if zero is acceptable output for GetMaxPostSize(), then I will simply update the test cases.

@jespino let me hear your thoughts on this.

Reply:
I don't see the need to handle that, It should be a very rare case and it should be logged by the store anyway, if we want to handle properly the error we can allow the error to raise from the `determineMaxPostSize` function. But I don't see the need to change it now.

Reply:
Well... I think you are right, is just safer to use 0 as invalid value, and return the `model.POST_MESSAGE_MAX_RUNES_V1`. Anyway is a new thing, because that code wasn't reachable before, as you mention.